boards/arm/stm32l562e_dk/stm32l562e_dk.yaml

```diff
@@ -10,5 +10,7 @@ supported:
   - i2c
   - lsm6dso
   - lptim
+  - spi
+  - arduino_spi
```

Comment:
Could you duplicate this information in both file stm32l562e_dk.yaml  stm32l562e_dk_ns.yaml
(secure and non secure) ?